src/CBT.AssemblyInfo/build/After.Microsoft.Common.targets

```diff
@@ -3,4 +3,17 @@
 
   <Import Project="$(MSBuildThisFileDirectory)AssemblyInfo.targets" Condition=" '$(EnableCBTAssemblyInfoGeneration)' == 'true' " />
 
+  <!-- Provide a deprication message to user with SDK projects. -->
+  <Target Name="DisableStockAssemblyInfoGeneration" BeforeTargets="GenerateAssemblyInfo" Condition=" '$(GeneratedAssemblyInfoFile)' != '' And '$(EnableCBTAssemblyInfoGeneration)' == 'true' And '$(EnableCBTAssemblyInfoDepricationCheck)' == 'true' ">
+    <PropertyGroup>
+      <_CBTDisableStockAssemblyInfoGenerationWarningMessage>
```

Comment:
I don't like that this is stored in a property because it will show up in a diagnostic log.  Is there a good reason to put it in a property?